src/index.js

```diff
@@ -2,6 +2,9 @@ import { create } from './create'
 import { run } from './run'
 
 const createWrapper = () => {
+  if ('undefined' === typeof window) {
```

Comment:
Are there any particular reason for use `undefined` on the left side. If not, can you please invert this order?